### PR TITLE
modulesync 4.0.0

### PR DESCRIPTION
**Pull request by bastelfreak**
modulesync 3.2.0

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+    strategy:
+      fail-fast: false
+      matrix:
+        setfile:
+          - value: fedora32-64{hostname=fedora32-64.example.com}
+            name: Fedora 32
+          - value: fedora33-64{hostname=fedora33-64.example.com}
+            name: Fedora 33
+          - value: centos6-64{hostname=centos6-64.example.com}
+            name: CentOS 6
```
Comment (ekohl) [inline .github/workflows/ci.yml:83]: CentOS 6 is going EOL in 2 weeks. Perhaps also time to drop it?

File: .sync.yml [hunk 2]
```diff
@@ -1,11 +1,4 @@
 ---
-.travis.yml:
-  secure: "YaM+1NaZ5PFn6rDg03uK/qJUfHPTTpk1wV0d8hy+O1g5xzt+v+Pn/ZzJ7hglBhEeWRNfBhcHWK9Wr/yghGHhZyjMhK4MXSIQnJjgSPVZFJM7PrH+IRzzwaCUT6tomnTdvbVWBr9R3BMgNUg2r1pvpAkFoDmWGcjJkwwQlDS1RnfkYmsf2rF5S0yOCqhdvTdsItHvpuXPMTnRtDhylvbQbp55BQXmHqDhFEv5JU9Rb1nXOaDPNqWeR+aaHp8IkSW/hiLn2giAVopek9gJVMeIqdlmXh5RQgCdPv/WOisMfsU/c3vc3z2EgdLZlWO7LkH2dh9H4129i2xLLuJB/CcpFyfWzNdqyu2aMf+yia0jkqy/QRZwjn+h4RKR+jmLWpGV+74m9z4EBkF0wdma0eiC7ZCY3E225E7UzX1o15lljWdQtd9Ou8sFWumoLRqnkNtSZR+kRHDdxem+M5pR2YK89iycCiywMCMw2uE6sdy51V/CIlZhwT7bXaogvld2WofSkm8iiaPVTyv3GiPmbK2YYlSh5//yqQz5Ggyaw78i1m7+4JW+ex2Saf8iJaU+29eWUT2/LjBHxljDXdTHuumsBjiigGl1MJg+w103Y8cnPLzQK1fxAeAolUrolm6UylAw0FepEI+4CtwkUJ0+QSL9b49lsN9xgc5EIahwtFnOZDo="
```
Comment (ekohl) [inline .sync.yml:3]: We don't have any replacement for this right now.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (bastelfreak) [reply .github/workflows/ci.yml:83]: fine by me. Did it in https://github.com/voxpupuli/puppet-borg/pull/81

**Comment by ekohl** [comment]
* Fedora 32 on Puppet 6 fails because it can't install `epel-release`
* F32 on Puppet 5 because that's unavailable
* F33 because beaker-hostgenerator doesn't support it
* EL8 can't install perl-TAP-Harness-Env

**Comment by ekohl** [comment]
As a test I pushed a commit that drops Puppet 5. I expect acceptance tests to drop it now.

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,114 @@
+name: CI
+
+on: pull_request
+
+env:
+  BUNDLE_PATH: vendor/bundle
+  BUNDLE_JOBS: 4
```
Comment (bastelfreak) [inline .github/workflows/ci.yml:7]: do we know how many cores github actions provide to us? Otherwise we could use something like `nproc` maybe?

Comment (ekohl) [reply .github/workflows/ci.yml:7]: I don't know, but with proper caching it isn't that much of a factor.

**Review by ekohl — CHANGES_REQUESTED** [review]
Needs an update now.

**Review by ekohl — CHANGES_REQUESTED** [review]
F32 on Puppet 5 because that's unavailable (expected)
F33 because beaker-hostgenerator doesn't support it (expected)
EL8 can't install perl-TAP-Harness-Env (previously untested)

The last one needs a fix.

**Comment by ekohl** [comment]
Looks like that broke the unit tests.

**Review by ekohl — APPROVED** [review]
I'd be fine with merging this if it only fails on F32 + P5 and F33.

**Pull request merged**